src/schemas/flow_node_instance.ts

```diff
@@ -14,6 +14,9 @@ export interface IFlowNodeInstanceAttributes {
   // Contains the association to the ProcessToken model.
   // Must be optional, otherwise this property will be expected in the attribute payload of `sequelize.define`.
   processTokens?: Array<ProcessToken>;
+  // The ID of the FNI that was executed before.
```

Comment:
I'd write FlowNodeInstance out in its full glory here :) 

```suggestion
  // The ID of the FlowNodeInstance that was executed before.
```

Reply:
Fair enough 😉